bip32/src/path.rs

```diff
@@ -41,6 +42,11 @@ fn encode_index(idx: u32, harden: char) -> String {
     s
 }
 
+/// Converts an raw index to hardened
+pub fn harden_index(index: u32) -> u32 {
+    index | (1 << 31)
```

Comment:
use the constant here instead of a shift

Reply:
fixed